ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -1105,6 +1105,7 @@ let start () =
   Metrics.with_metrics metrics run
 
 let run ~read_dot_merlin () =
+  Merlin_utils.Lib_config.set_program_name "OCaml-LSP";
```

Comment:
How about just using the binary name? `ocamllsp`.

Reply:
fixed